packages/pigeon/platform_tests/test_plugin/ios/Classes/TestPlugin.swift

```diff
@@ -1214,14 +1217,63 @@ public class TestPluginWithSuffix: HostSmallApi {
 
 }
 
+class SendInts: StreamIntsStreamHandler {
+  var timerActive = false
+
+  override func onListen(withArguments arguments: Any?, sink: PigeonEventSink<Int64>) {
+    var count: Int64 = 0
+    if !timerActive {
+      timerActive = true
+      Timer.scheduledTimer(withTimeInterval: 0.01, repeats: true) { _ in
```

Comment:
These timers never end, so it'll spam the runloop with no-ops for as long as the process exists. I think you probably want `DispatchQueue.main.async` instead.